xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -13217,8 +13231,7 @@ glusterd_enable_default_options(glusterd_volinfo_t *volinfo, char *option)
     }
 
     if ((conf->op_version >= GD_OP_VERSION_9_0) &&
-        (volinfo->status == GLUSTERD_STATUS_NONE) &&
-        (volinfo->type == GF_CLUSTER_TYPE_REPLICATE)) {
```

Comment:
Why has this been removed ?

Reply:
it was removed by mistake during resolve conflict issue.